src/lessons/20_security/security.slides.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Security</title>
+    <meta name="date" content="2021-06-15" />
+    <meta
+      name="description"
+      content="
+      - Call stack
+      - Security
+      - CORS, XSS, CRSF, HTTPS
+    "
+    />
+  </head>
+
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <section>
+          <p>Hey! 👋🏼</p>
+          <h1 class="r-fit-text">Security</h1>
+          <p>Redi School Munich - Spring 2021</p>
+        </section>
+        <section>
+            <h2>Recap!</h2>
+            <ul>
+                <li class="fragment">Kahoot!</li>
+                <li class="fragment">Callbacks</li>
+                <li class="fragment">Promises</li>
+            </ul>
+        </section>
+        <section>
+            <h3>JavaScript is a single threaded language</h3>
+            <p class="fragment">Only one thing can be run at a time</p>
+            <p class="fragment">There is something called the call stack</p>
+        </section>
+        <section>
+            <h3>The callstack can only ever run one thing at a time</h3>
+        </section>
+        <section>
+            <h1>Security!</h1>
+            <img src="./images/hackerman.gif" alt="hackerman gif">
+        </section>
+        <section>
+          <h2>Security</h2>
+          <p class="fragment">Javascript is public!</p>
```

Comment:
I'd rephrase this. Maybe "Javascript is a client-side language" or "Javascript code can be inspected by users"

Reply:
fixed!